### ISSUE TITLE
Error installing SCPA

### ISSUE THREAD

**Issue by nikolasgiannak** [jackbibby1/SCPA#75]
**What's going on**
I am trying to install the SCPA PACKAGE ( R version  4.3.3) without success,It seems the problem is "multicross" package, which is a dependency.

**Code to replicate the issue**

More specifically, I have tried the following as suggested previously:
devtools::install_version("crossmatch", version = "1.3.1", repos = "http://cran.us.r-project.org") #this worked

devtools::install_version("multicross", version = "2.1.0", repos = "http://cran.us.r-project.org")

devtools::install_github("jackbibby1/SCPA")

This is the error I got:
ERROR: dependency ‘multicross’ is not available for package ‘SCPA’
* removing ‘/Library/Frameworks/R.framework/Versions/4.3-x86_64/Resources/library/SCPA’
Warning messages:
1: package ‘multicross’ is not available for this version of R

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages 
2: In i.p(...) :
  installation of package ‘/var/folders/gf/08xrpmbx5bbcnwynpmnzr5zr0000gn/T//RtmpGnYbsQ/file1b343a68f4ad/SCPA_1.6.2.tar.gz’ had non-zero exit status
>

**Comment by jackbibby1** [comment]
Hi,

Did the `devtools::install_version("multicross", version = "2.1.0", repos = "http://cran.us.r-project.org/")` run fine?

Jack

**Comment by nikolasgiannak** [comment]
After this:

devtools::install_version("multicross", version = "2.1.0", repos = "http://cran.us.r-project.org")

It gives me this error:

Warning: unable to access index for repository http://cran.us.r-project.org/src/contrib:
  cannot open URL 'http://cran.us.r-project.org/src/contrib/PACKAGES'
Error in download_version_url(package, version, repos, type) : 
  couldn't find package 'multicross'

**Comment by jack-bibby** [comment]
Hi,

It looks like it might be issue with the CRAN mirror. Can you try this:

```
devtools::install_version("multicross", version = "2.1.0", repos = "http://cloud.r-project.org")
```

Jack

**Comment by jackbibby1** [comment]
Hi,

It looks like it might be issue with the CRAN mirror. Can you try this:

```
devtools::install_version("multicross", version = "2.1.0", repos = "http://cloud.r-project.org")
```

Jack

**Comment by nikolasgiannak** [comment]
It's the version of R that I am using that causes the problem.

ERROR: dependency ‘multicross’ is not available for package ‘SCPA’
* removing ‘/Library/Frameworks/R.framework/Versions/4.3-x86_64/Resources/library/SCPA’
Warning messages:
1: package ‘multicross’ is not available for this version of R

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages 
2: In i.p(...) :
  installation of package ‘/var/folders/gf/08xrpmbx5bbcnwynpmnzr5zr0000gn/T//Rtmp5ttwf1/file26b1106cf53/SCPA_1.6.2.tar.gz’ had non-zero exit status

But does it mean that for this version of R or later, users won't be able to effectively install SCPA, which is very informative in terms go biological interpretation of the data?

**Comment by jackbibby1** [comment]
Your R version shouldn't be an issue for installing multicross. I'm on R v4.4 and things are looking fine. Did you try the alternative mirror in my previous reply? i.e. the different `repos = ` argument?

```
devtools::install_version("multicross", version = "2.1.0", repos = "http://cloud.r-project.org")
```

**Comment by nikolasgiannak** [comment]
Problem solved.
Thanks for the tips.

What I did was to uninstall crossmatch and restart R. Then, 
1) I installed again crossmatch from the cloud Cran mirror like this:
devtools::install_version("crossmatch", version = "1.3.1", repos = "http://cloud.r-project.org")

2) Installed multi cross as indicated:
devtools::install_version("multicross", version = "2.1.0", repos = "http://cloud.r-project.org")

3) Loaded the two libraries
library(crossmatch)
library(multi cross)

4) Installed SCPA and loaded the library
devtools::install_github("jackbibby1/SCPA")
library(SCPA)

Cheers!
